AEPServices/Sources/ui/alert/AlertMessaging.swift

```diff
@@ -12,16 +12,17 @@
 
 import Foundation
 
-/// UI service interface defining a fullscreen message
-public protocol FullScreenMessageUiInterface {
+/// Fullscreen message lifecycle event listener
+@objc(AEPAlertMessaging) public protocol AlertMessaging {
+    /// Invoked on positive button clicks
+    func OnPositiveResponse()
```

Comment:
The first letter of the function name should be lowercase.
`onPositiveResponse`